content/client-server-api/_index.md

```diff
@@ -2540,6 +2540,9 @@ return a standard error response of the form:
 The `retry_after_ms` key SHOULD be included to tell the client how long
 they have to wait in milliseconds before they can try again.
 
+The `Retry-After` HTTP header MAY be specified by the server. When this header
+is set, it SHOULD be preferred by clients over the `retry_after_ms` value.
```

Comment:
The MSC also says that the header should be on *any* 429 response, not just `M_LIMIT_EXCEEDED`. I'm not sure if, in practice, that makes any difference, but it feels like the requirement to use `Retry-After` belongs under the "API standards" section rather than buried here where nobody will read it. (Possibly an *informative* reminder in this "Security" section could be useful too, but I feel like the *normative* requirement doesn't belong here.)

Reply:
I've moved the section to API standards, agree it makes sense. The security section only had a bit on ratelimiting, so I've just purged the section rather than leaving it empty.